distribution/lib/Standard/Base/0.0.0-dev/src/Error/Common.enso

```diff
@@ -321,7 +321,7 @@ type Panic
             Caught_Panic_Data _ internal_original_exception -> internal_original_exception
             throwable -> throwable
         prim_stack = Panic.primitive_get_attached_stack_trace throwable
-        stack_with_prims = Vector.Vector_Data prim_stack
+        stack_with_prims = Vector.from_array prim_stack
```

Comment:
Again (sorry if too repetitive) - I think we should revisit and ensure we are using the non-copying constructors in places where we guarantee immutability of the original array. There's many such places. Ofc first we need to discuss if there's anything stopping us from doing so: is there?

Reply:
Replaced with `from_polyglot_array` in ce6aa5cddb9e6